internal/cryptsetup/cryptsetup.go

```diff
@@ -64,6 +64,9 @@ func (d *Device) Format(ctx context.Context) error {
 		"--cipher=aes-xts-plain64",              // Use AES-XTS cipher.
 		"--pbkdf=argon2id",                      // Use Argon2id as the key derivation function.
 		"--pbkdf-memory=10240",                  // Memory usage for Argon2i, limit to 10 MiB so it won't fail in low-memory pods.
+		"--integrity=hmac-sha256",               // Use HMAC-SHA256 for integrity protection via dm-verity.
```

Comment:
I'm a bit confused, isn't this using dm-integrity, not dm-verity? AFAIK dm-verity is only for read-only partitions, right?